xcq-extension/Cargo.toml

```diff
@@ -13,10 +13,8 @@ scale-info = { workspace = true }
 xcq-executor = { workspace = true }
 impl-trait-for-tuples = "0.2.2"
 tracing = { workspace = true }
+xcq-extension-procedural = { path = "procedural" }
```

Comment:
use workspace deps to be consistent

Reply:
I supposed to make it only as a dependency of `xcq-extension` but unseen to outer world.